src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/DiagnosticDescriptorCreationAnalyzer_ReleaseTracking.cs

```diff
@@ -152,7 +152,7 @@ public sealed partial class DiagnosticDescriptorCreationAnalyzer
             category: DiagnosticCategory.MicrosoftCodeAnalysisReleaseTracking,
             defaultSeverity: DiagnosticSeverity.Warning,
             isEnabledByDefault: true,
-            description: CodeAnalysisDiagnosticsResources.EnableAnalyzerReleaseTrackingRuleDescription,
+            description: new LocalizableResourceString(nameof(CodeAnalysisDiagnosticsResources.EnableAnalyzerReleaseTrackingRuleDescription), CodeAnalysisDiagnosticsResources.ResourceManager, typeof(CodeAnalysisDiagnosticsResources)),
```

Comment:
Shouldn't these changes have led to updates to the md/sarif files? The changes by themselves look good, but am wondering if there are more changes in the generated files that need to be included.. Doesn't block this PR though..

Reply:
@mavasani I run pack to confirm, but no this doesn't lead to any updates in generated files. The description is still the same.